testing/infrastructure/InfrastructureComponentShape.ttl

```diff
@@ -27,34 +27,35 @@ shapes:InfrastructureComponentShape
   sh:property [
   	a sh:PropertyShape ;
   	sh:path ids:maintainer ;
-		#sh:or (
-	  #  [ sh:class ids:Participant ; ]
-	  #  [
-		sh:nodeKind sh:IRI ;# ]
-		#) ;
+		sh:or (
+	    [ sh:class ids:Participant ; sh:order 2 ; ]
+	    [	sh:nodeKind sh:IRI ; sh:order 1 ; ]
+		) ;
     sh:minCount 1 ;
+  	sh:maxCount 1 ;
   	sh:severity sh:Violation ;
   	sh:message "<https://raw.githubusercontent.com/International-Data-Spaces-Association/InformationModel/master/testing/infrastructure/InfrastructureComponentShape.ttl> (InfrastructureComponentShape): An ids:maintainer property must have at least one point from an ids:InfrastructureComponent to an ids:Participant."@en ;
   ] ;
 
   sh:property [
     a sh:PropertyShape ;
     sh:path ids:curator ;
-		#sh:or (
-	  #  [ sh:class ids:Participant ; ]
-	  #  [
-		sh:nodeKind sh:IRI ; #]
-		#) ;
-    sh:minCount 1 ;
+		sh:or (
+	    [ sh:class ids:Participant ; sh:order 2 ; ]
+	    [ sh:nodeKind sh:IRI ; sh:order 1 ; ]
+		) ;
+    #sh:minCount 1 ;
```

Comment:
Why is this commented-out? It was not before.

Reply:
I tried one thing with our CodeGen Tool and forgot to undo my test. Will fix it